src/components/RadioButton/index.js

```diff
@@ -0,0 +1,72 @@
+import { Component } from 'pet-dex-utilities';
+import './index.scss';
+
+const events = ['change', 'value:change', 'text:change', 'name:change'];
```

Comment:
Como vai precisar do método para desabilitar, então precisamos adicionar ele

```suggestion
const events = ['change', 'value:change', 'text:change', 'name:change', 'disable'];
```